x/datapool/types/keys.go

```diff
@@ -51,3 +53,9 @@ func GetKeyPrefixDataValidateCertByRound(poolID, round uint64) []byte {
 func GetKeyPrefixDataValidateCert(poolID, round uint64, dataHash []byte) []byte {
 	return append(GetKeyPrefixDataValidateCertByRound(poolID, round), dataHash...)
 }
+
+func GetKeyPrefixNFTRedeemReceipt(poolID, round uint64, redeemer sdk.AccAddress) []byte {
+	poolAppend := append(KeyPrefixNFTRedeemReceipts, sdk.Uint64ToBigEndian(poolID)...)
+	roundAppend := append(poolAppend, sdk.Uint64ToBigEndian(round)...)
```

Comment:
My guess is that a value with poolID 1 and round 11 and a value with poolID 11 and round 1 will come out the same.
How about using CombineKey like dataDeal?
https://github.com/medibloc/panacea-core/blob/master/x/datadeal/types/keys.go#L48

Reply:
Thanks, the key will be changed like [this](https://github.com/medibloc/panacea-core/blob/ft/280/redeem_data_pass_cli/x/datapool/types/keys.go), too. Because of the pagination and the request of data validator for data pass redeem receipts.

Reply:
I got it. I'll check back in the next PR for changes.